sdk/eventhub/event-hubs/src/partitionProcessor.ts

```diff
@@ -81,6 +87,23 @@ export class PartitionProcessor {
     this._lastEnqueuedEventInfo = lastEnqueuedEventInfo;
   }
 
+  /**
+   * @property The fully qualified namespace from where the current partition is being processed. It is set by the `EventProcessor`
+   * @readonly
+   */
+  public get fullyQualifiedNamespace() {
+    return this._consumerGroupName!;
```

Comment:
```suggestion
    return this._fullyQualifiedNamespace!;
```

Reply:
Nice catch! We don't have a test for this, will add a test case.